flexget/plugins/modify/list_clear.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginError
+
+log = logging.getLogger('list_clear')
+
+
+class ListClear(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'what': {'type': 'array', 'items':
+                {'allOf': [
+                    {'$ref': '/schema/plugins?group=list'},
+                    {'maxProperties': 1,
+                     'error_maxProperties': 'Plugin options within list_clear plugin must be indented '
+                                            '2 more spaces than the first letter of the plugin name.',
+                     'minProperties': 1}]}},
+            'phase': {'type': 'string', 'enum': plugin.task_phases, 'default': 'exit'}
+        },
+        'required': ['what']
+    }
+
+    @plugin.priority(255)
+    def on_task_start(self, task, config):
+        for item in config['what']:
+            for plugin_name, plugin_config in item.items():
+                try:
+                    thelist = plugin.get_plugin_by_name(plugin_name).instance.get_list(plugin_config)
+                except AttributeError:
+                    raise PluginError('Plugin %s does not support list interface' % plugin_name)
+                if thelist.immutable:
+                    raise plugin.PluginError(thelist.immutable)
+                if config['phase'] == task.current_phase:
+                    if task.manager.options.test:
```

Comment:
since test mode creates a temp db, anything that is a part of that temp db only should persist in that scope, since it can affect other operations. Al plugins that rely on our db still work, so why not `list_clear`?